README.rst

```diff
@@ -41,7 +41,7 @@ Installation
         "post_office",
     )
 
-* Run ``syncdb`` for Django <= 1.8::
+* Run ``syncdb`` for Django == 1.8::
```

Comment:
Yes, I agree with you, I just noticed this because I've been using Django 1.7 in all my projects and have not started a new one with Django 1.8 yet. Thanks for the correction @yprez , I will now commit a new one which removes ``syncdb``